docs/chronicle/rancher.md

```diff
@@ -0,0 +1,123 @@
+In our example, this will be the local cluster
+
+From the left menu, select _Apps_ and then _Charts_.
+Choose the Chronicle chart from the list of partner charts:
+
+![Partner Charts](../images/chronicle/rancher/partner-charts.png)
+
+This will take you to the following screen,
+click on the _Install_ button on the top right of the page:
+
+![Chart Details](../images/chronicle/rancher/chart-details.png)
+
+Here, you will need to specify the _namespace_ for your Chronicle
+installation. In our example, we will use the existing `chronicle` namespace:
+
+!!!Note
+    If the namespace doesn't exist, the installation process will automatically
+    create this for you.
+
+Now, click the _Next_ button on the bottom right of the page:
+
+![Install Namespace](../images/chronicle/rancher/install-namespace.png)
+
+On this screen you can configure your Chronicle installation. On the left hand
+side, you will find three options:
+
+* **Chronicle Settings** - Here you can configure the Chronicle image and tag
+  that you want to use. We will leave these as the defaults for now.
+  You can also enable a development GraphQL playground, however in this example
+  we will leave this disabled, and use the Altair GraphQL client instead.
+* **Ingress Settings** - If you'd like to enable an ingress for Chronicle,
+  you can specify this here. This is optional.
+* **Database Settings** - If you'd like to use an external Postgres database,
+  you can specify this here. This is also optional.
+
+Click the _Install_ button on the bottom right of the page.
+
+Rancher will now install Chronicle on your local cluster. It may take a few
+minutes for the Chronicle images to be pulled down, and for the
+underlying Sawtooth network to be deployed.
+
+![Installing Chronicle](../images/chronicle/rancher/installing.png)
+
+We will now switch to a local terminal window to test our Chronicle install.
+
+Once you've opened a local terminal, start by confirming that you can connect to
+your Kubernetes cluster using `kubectl` by running this command:
+
+```bash
+kubectl get pods -n chronicle
+```
+
+![Check kubectl](../images/chronicle/rancher/check-kubectl.png)
+
+We now need to set up a port forward so that we can access the Chronicle api
+on our local machine.
+Run the following command to set up a port forward:
+
+```bash
+kubectl port-forward chronicle-chronicle-0 9982:9982 -n chronicle
+```
+
+This will set up a port forward to your Sextant install, and make it accessible
+on your local machine:
+
+![Port forward](../images/chronicle/rancher/port-forward.png)
+
+You can now access the Chronicle GraphQL API using the following URL at [http://127.0.0.1:9982](http://127.0.0.1:9982)
+
+We will use the [Altair GraphQL client](https://github.com/altair-graphql/altair)
+to test our Chronicle install in the browser
+
+Open the Altair GraphQL client in your browser, and paste in the our
+Chronicle API URL following URL `http://127.0.0.1:9982`
```

Comment:
```suggestion
Open the Altair GraphQL client in your browser, and paste in the following URL
`http://127.0.0.1:9982`
```

Reply:
@suchapalaver thanks, I've corrected this